docs/make.jl

```diff
@@ -9,8 +9,16 @@ makedocs(
         "Introduction" => "index.md",
         "Installation Guide" => "installation.md",
         "Quick Start Guide" => "quickstart.md",
-        "Expressions and Constraints" => "refexpr.md",
-        "Nonlinear Modeling" => "nlp.md"
+        "Concepts" => "concepts.md",
+        "Variables" => "variables.md",
+        "Expressions" => "expressions.md",
+        "Constraints" => "constraints.md",
+        "Solvers" => "solvers.md",
+        "Style Guide" => "style.md",
+        "Extensions" => "extensions.md",
+        "Nonlinear Modeling" => "nlp.md",
```

Comment:
Why is "Nonlinear Modeling" (who is meant for user) placed after "Style Guide" and "Extensions" (who is meant more for developers) ?